commands/install.js

```diff
@@ -53,7 +53,14 @@ async function install (argv, arg1, arg2, arg3) {
   L(line('|➔', chalk`Install this module? {${confirm}}`, COLORS.yellow))
   L()
 
-  const choice = (await queryReadline(chalk`{${COLORS.light1} > }`)).trim().toLowerCase()
+  let choice
+  if (argv.force) {
+    choice = 'y'
+  } else if (process.stdin.isTTY) {
+    choice = (await queryReadline(chalk`{${COLORS.light1} > }`)).trim().toLowerCase()
+  } else {
+    choice = good ? 'y' : 'n'
+  }
```

Comment:
could be a `const`:

```js
const choice = argv.force
  ? ...
  : ...
```

Reply:
Oh, true. Cases might be a bit long tho.

Reply:
:shrug:

```js
const choice = argv.force
  ? 'y'
  : process.stdin.isTTY
    ? (await queryReadline(chalk`{${COLORS.light1} > }`)).trim().toLowerCase()
    : good ? 'y' : 'n'
```